hardhat.config.ts

```diff
@@ -7,6 +7,7 @@ import "@nomiclabs/hardhat-waffle";
 import "hardhat-typechain";
 import "solidity-coverage";
 import "hardhat-deploy";
+import "hardhat-gas-reporter"
```

Comment:
Let's separate all of these gas-reporter updates into another PR. If we want to include these it should be a conscious decision. Don't forget to rerun `yarn install` to push the correct `yarn.lock`